blueprints/app/files/testem.js

```diff
@@ -13,7 +13,10 @@ module.exports = {
       args: [
         // --no-sandbox is needed when running Chrome inside a container
         process.env.TRAVIS ? '--no-sandbox' : null,
-
+        '--disable-gpu',
+        '--disable-dev-shm-usage',
+        '--disable-software-rasterizer',
+        '--mute-audio',
         '--disable-gpu',
         '--headless',
```

Comment:
minor thing: I like to see `--headless` as the first argument, since it makes it clear why the rest of the arguments exists.